quilt/tools/command.py

```diff
@@ -148,6 +148,12 @@ def _parse_package(name):
             raise ValueError
     except ValueError:
         raise CommandException("Specify package as owner/package_name.")
+
+    try:
+        PackageStore.check_name(owner, pkg)
+    except StoreException as ex:
+        raise CommandException(str(ex))
```

Comment:
The error message stays the same ("Invalid package name: ...") - the only difference is, the `main` method catches `CommandException` and just prints out the message so you don't see the stack trace.